sunpy/database/tests/test_attrs.py

```diff
@@ -407,6 +407,13 @@ def test_walker_create_vso_instrument(vso_session):
     entries = walker.create(vso.attrs.Instrument('RHESSI'), vso_session)
     expected = [
         tables.DatabaseEntry(id=1, source=u'RHESSI', provider=u'LSSP',
+                             physobs=u'intensity',
```

Comment:
this is causing a test to fail.

Reply:
It looks like the second and third entries in `expected` are the same? Should the last one be removed? Or should we actually expect 3 entries from `vso_session` for RHESSI?

Reply:
I'll fix this tonight